lib/vanagon/component/source/http.rb

```diff
@@ -65,12 +70,12 @@ def file
         # Verify the downloaded file matches the provided sum
         #
         # @raise [RuntimeError] an exception is raised if the sum does not match the sum of the file
-        def verify
+        def verify # rubocop:disable Metrics/AbcSize
           puts "Verifying file: #{file} against sum: '#{sum}'"
-          actual = get_md5sum(File.join(workdir, file))
+          actual = get_sum(File.join(workdir, file), sum_type)
           return true if sum == actual
 
-          fail "Unable to verify '#{File.join(workdir, file)}': md5sum mismatch (expected '#{sum}', got '#{actual}'')"
+          fail "Unable to verify '#{File.join(workdir, file)}': #{sum_type} mismatch (expected '#{sum}', got '#{actual}'')"
```

Comment:
It's a prexisting typo, that I probably made, but can you clean up the extra single quote in `'#{actual}''` while you're in here? As a favor to me, maybe? Please & thank you.